src/modules/Dashboard/components/DownloadInfoModal/__tests__/DownloadInfoModal.jsx

```diff
@@ -0,0 +1,87 @@
+import { configure, mount } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import * as React from "react";
+import { Modal } from "react-bootstrap";
+import { cleanup } from "@testing-library/react";
+
+import DownloadInfoModal from "../DownloadInfoModal";
+
+const failedText = "The build is failed. Try to build it again.";
```

Comment:
Import `failedText` and `waitText` from 'src/modules/Dashboard/components/DownloadInfoModal/DownloadInfoModal.jsx'.